tests/helpers.py

```diff
@@ -1,81 +1,12 @@
-# Standard Imports
-from unittest.mock import ANY, Mock
-
-# ThirdParty Imports
 import asyncio
 
-# Internal Imports
-from slack.web.slack_response import SlackResponse
-
-
-def fake_req_args(headers=ANY, data=ANY, params=ANY, json=ANY):
-    req_args = {
-        "headers": headers,
-        "data": data,
-        "params": params,
-        "json": json,
-        "ssl": ANY,
-        "proxy": ANY,
-        "auth": ANY,
-    }
-    return req_args
-
-
-def fake_send_req_args(headers=ANY, data=ANY, params=ANY, json=ANY):
-    req_args = {
-        "headers": headers,
-        "data": data,
-        "params": params,
-        "json": json,
-        "ssl": ANY,
-        "proxy": ANY,
-        "files": ANY,
-        "auth": ANY,
-    }
-    return req_args
-
-
-def mock_rtm_response():
-    coro = Mock(name="RTMResponse")
-    data = {
-        "client": ANY,
-        "http_verb": ANY,
-        "api_url": ANY,
-        "req_args": ANY,
-        "data": {
-            "ok": True,
-            "url": "ws://localhost:8765",
-            "self": {"id": "U01234ABC", "name": "robotoverlord"},
-            "team": {
-                "domain": "exampledomain",
-                "id": "T123450FP",
-                "name": "ExampleName",
-            },
-        },
-        "headers": ANY,
-        "status_code": 200,
-    }
-    coro.return_value = SlackResponse(**data)
-    corofunc = Mock(name="mock_rtm_response", side_effect=asyncio.coroutine(coro))
```

Comment:
I removed some existing mock utilities depending on asyncio. The dependency caused the difficulties for detecting potential concurrency issues when `run_async=False`.